README.md

```diff
@@ -32,114 +34,9 @@ mkdir build && cd build
 cmake .. -DCMAKE_TOOLCHAIN_FILE=../cmake/arm-none-eabi.cmake -DPLATFORM_NAME=nrf52
 make -j12
 ```
+This will build the application of OpenThread command line in `build/ot_cli_nrf52840.hex`, you can use the command line application by flashing the binary with `nrfjprog`([Download](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF5-Command-Line-Tools)) and connecting to the serial port of NRF52840 dev kit.
```

Comment:
Suggestion:

"This will build the CLI test application in `build/ot_cli_nrf52840.hex`. You can flash the binary with `nrfjprog`([Download](https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF5-Command-Line-Tools)) and connecting to the nRF52840 DK serial port."